packages/tuono-router/src/components/RouterContext.tsx

```diff
@@ -1,13 +1,29 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
```

Comment:
Style: `useState` and `useEffect` are imported via named import while all context functions are prefixed with `React.`. 
I think we should be more consistent. 
I usually prefer importing `React` and prefix all with `React.` (also properties which starts with `React`. It's redundant, but at least the style is consistent on all files).
There are types like `ElementType` or handlers (like `MouseEventHandler`, ...) that cannot be associated with React when used without `React.`

(I can take care of this in a separate PR)

Reply:
I'd always use named imports.

Reply:
- #259